pandas/tests/series/test_constructors.py

```diff
@@ -725,3 +725,30 @@ def f():
         self.assertEqual(s.dtype, 'timedelta64[ns]')
         s = Series([pd.NaT, np.nan, '1 Day'])
         self.assertEqual(s.dtype, 'timedelta64[ns]')
+
```

Comment:
you can do these in 2 tests. _hashable, and _unhashable. just put a loop in each one.